src/processors/chain/ProcessorChainActionHelper.h

```diff
@@ -11,6 +11,7 @@ class ProcessorChainActionHelper : private HighResolutionTimer
     void addProcessor (BaseProcessor::Ptr newProc);
     void removeProcessor (BaseProcessor* procToRemove);
     void replaceProcessor (BaseProcessor::Ptr newProc, BaseProcessor* procToReplace);
+    void addProcessorFromCableClick (BaseProcessor::Ptr newProc, BaseProcessor* startProc, BaseProcessor* endProc);
```

Comment:
I think for this method (and the other similar ones) I'd prefer a name like `replaceConnectionWithProcessor()`, just in case we later choose to allow another interface for interacting with this behaviour rather than clicking on the cables.

Reply:
Good idea